spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/actuator/tracing.adoc

```diff
@@ -88,9 +88,10 @@ logging:
   pattern:
     correlation: "[${spring.application.name:},%X{traceId:-},%X{spanId:-}] "
   include-application-name: false
+  include-application-group: false
 ----
 
-NOTE: In the example above, configprop:logging.include-application-name[] is set to `false` to avoid the application name being duplicated in the log messages (configprop:logging.pattern.correlation[] already contains it).
+NOTE: In the example above, configprop:logging.include-application-name[] and configprop:logging.include-application-group[] is set to `false` to avoid the application name being duplicated in the log messages (configprop:logging.pattern.correlation[] already contains it).
```

Comment:
`include-application-group` does not need to be set to `false` here as the preceding example of a custom correlation pattern doesn't include `spring.application.group`. The example shouldn't be changed as it matches Spring Cloud Sleuth's behavior. I think the change on this line should be reverted instead.